dockerfiles/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # FROM directive instructing base image to build upon
 # This could be used as a base instead: 
 # https://hub.docker.com/r/nikolaik/python-nodejs
-FROM python:3.8-slim
+FROM python:3.10-slim-bookworm
```

Comment:
This is version used by the MyLA. ROHQ uses `python:3.10-slim-bullseye`. Maybe we can adopt the same model among all T&L apps

Reply:
Agreed, bullseye [is no longer supported](https://endoflife.date/debian). MyLA was [upgraded to bookworm](https://github.com/tl-its-umich-edu/my-learning-analytics/issues/1513) last year, so an issue should be created to upgrade ROHQ and any place else this is noticed.